core/com.b2international.snowowl.datastore/src/com/b2international/snowowl/core/validation/ValidateRequest.java

```diff
@@ -274,5 +275,10 @@ public IssuesToPersist(String ruleId, Collection<?> evaluationResult) {
 		}
 		
 	}
+
+	@Override
+	public Permission getPermission() {
+		return new Permission(Permission.BROWSE, Permission.ALL);
```

Comment:
Operation is debatable. A validation will "write" a bunch of stuff, and remove the earlier set of results, so it might need `Permission.EDIT`.